addons/common/XEH_preInit.sqf

```diff
@@ -6,6 +6,13 @@ GVAR(ACE_Loaded) = isClass (configFile >> "CfgPatches" >> "ace_common");
 GVAR(CUP_Loaded) = isClass (configFile >> "CfgPatches" >> "cup_common"); // Probably something else, not "cup_common"
 GVAR(RHS_Loaded) = isClass (configFile >> "CfgPatches" >> "rhs_common"); // Same here
 
+// Prepare all location types for getNearestLocation function
+GVAR(allLocationTypes) = [];
+QUOTE(GVAR(allLocationTypes) pushBack configName _x) configClasses (configFile >> "CfgLocationTypes");
```

Comment:
`configClasses` accepts condition in string as left operator, shouldn't it be:
```sqf
GVAR(allLocationTypes) = ("true" configClasses (configFile >> "CfgLocationTypes")) apply {configName _x};
```
?
or 
```sqf
QUOTE(GVAR(allLocationTypes) pushBack configName _x; true) configClasses (configFile >> "CfgLocationTypes");
```
if you want to abuse that a little bit.

Reply:
It is from wiki example. I tested it and it worked. You thought it can not work? Then well it does. 😛 Or do you mean, although it works, it should not be done like this?

Reply:
The left hand argument is supposed to be a condition code as string so it should return an \<BOOL\>, `pushBack` returns index of added element, I don't see an example with `pushBack` on wiki and all of them return bool there. But if it works (even just by a luck) 🤷‍♂ 

At least consider adding `; true` so the return type is as expected.

Reply:
There is one, on [nearestLocations](https://community.bistudio.com/wiki/nearestLocations) page.

I will change it to
```SQF
GVAR(allLocationTypes) = ("true" configClasses (configFile >> "CfgLocationTypes")) apply {configName _x};
```
as it looks much clearer, better and is probably faster. The wiki example could also be modified to use this method as the one with `pushBack` is kinda _hacky_.